contracts/PartyBid.sol

```diff
@@ -462,83 +564,40 @@ contract PartyBid is ReentrancyGuardUpgradeable, ERC721HolderUpgradeable {
      * @notice Upon winning the auction, transfer the NFT
      * to fractional.art vault & mint fractional ERC-20 tokens
      */
-    function _fractionalizeNFT(uint256 _totalSpent) internal {
+    function _fractionalizeNFT() internal {
         // approve fractionalized NFT Factory to withdraw NFT
         IERC721Metadata(nftContract).approve(tokenVaultFactory, tokenId);
+        // PartyBid "votes" for a reserve price on Fractional
+        // equal to 2x the winning bid
+        uint256 _listPrice = RESALE_MULTIPLIER * highestBid;
+        // users receive tokens at a rate of 1:TOKEN_SCALE for each ETH they contributed that was ultimately spent
+        // partyDAO receives a percentage of the total token supply equivalent to TOKEN_FEE_BASIS_POINTS
+        // partyHost receives a percentage of the total token supply equivalent to partyHostBasisPoints
+        (uint256 _tokenSupply, uint256 _tokenFee, uint256 _partyHostAmount) = _getTokenInflationAmounts(totalSpent);
```

Comment:
no biggie but i kinda expected this to be called `_partyDAOAmount` instead of `_tokenFee` to match the params returned by the fn

Reply:
yep agree